File: safe_place/v1/by_coords/python/safe_place.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import ResponseError, Client

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    Returns safety information for a location in Barcelona based a geolocation coordinates
    '''
    response = amadeus.safety.safety_rated_locations.get(latitude=41.397158, longitude=2.160873)
    print(response.data)
except ResponseError as error:
    raise error

File: safe_place/v1/get/safety_rated_locations_safety_rated_locationId/iOS SDK/SafePlace.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.safety.safetyRatedLocation(locationId: "Q930402753").get(onCompletion: {
    result in
    switch result {
    case .success(let response):
        print(response.data)
    case .failure(let error):
        print(error.localizedDescription)
    }
})

